lib/tests/unittests/SlashRemoverTest.cc

```diff
@@ -0,0 +1,29 @@
+#include <drogon/drogon_test.h>
+#include <../../lib/src/SlashRemover.cc>
```

Comment:
If anyone has a better solution than this line, let me know

Reply:
LGTM